src/scripts/main.js

```diff
@@ -1,3 +1,354 @@
 'use strict';
 
-// write code here
+const mainPage = document.querySelector('body');
+
+let table = document.querySelector('tbody');
+
+const header = document.querySelector('thead').lastElementChild;
+
+let rows = [];
+
+const successTitle = 'Success';
+const successMessage = 'New employee record has been added successfully';
+const errorTitle = 'Error';
+let errorMessage = '';
+
+function sortAsc(i) {
+  rows = [...document.querySelector('tbody').children].sort((a, b) =>
+    a.children[i].innerHTML
+      .localeCompare(b.children[i].innerHTML));
+
+  return rows;
+}
+
+function sortDesc(i) {
+  rows = [...document.querySelector('tbody').children]
+    .sort((a, b) => b.children[i].innerHTML
+      .localeCompare(a.children[i].innerHTML));
+
+  return rows;
+}
+
+function sortAscAge(i) {
+  rows = [...document.querySelector('tbody').children]
+    .sort((a, b) =>
+      a.children[i].innerHTML - b.children[i].innerHTML);
+
+  return rows;
+}
+
+function sortDescAge(i) {
+  rows = [...document.querySelector('tbody').children]
+    .sort((a, b) =>
+      b.children[i].innerHTML - a.children[i].innerHTML);
+
+  return rows;
+}
+
+function sortAscSalary(i) {
```

Comment:
handle all types of sorting inside your main sort function, don't create a separate method for a separate case, your method should be reusable and extendable, right now you copy/paste a lot of code.
 firstly decide it's desc or Asc sorting, after that use the `sort` method, and inside the `sort` method, depending on which field you working with, use different callbacks